### PR TITLE
Updated german translation

### PR DESCRIPTION
**Pull request by Ablu**

### CODE REVIEW

File: translations/tiled_de.ts [hunk 1]
```diff
@@ -3181,12 +3186,16 @@ Bitte Format angeben.</translation>
...
 &lt;p&gt;Every little bit helps. Tiled has a lot of users and if each would contribute a small donation each month I will have time to make sure Tiled keeps getting better.&lt;/p&gt;
 &lt;p&gt;&lt;i&gt;Thorbj&amp;oslash;rn Lindeijer&lt;/i&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
+&lt;h3&gt;Mit Deiner Hilfe kann ich weitermachen Tiled zu verbessern!&lt;/h3&gt;
+&lt;p&gt;Bitte überlege mich zu unterstützen. Deine Unterstützung würde einen großen Unterschied für mich als Haupt-Entwickler von Tiled machen. Ich könnte weniger Zeit damit verbringen Geld anderweitig zu verdienen und somit mehr Zeit in Tiled selbst stecken.&lt;/p&gt;
+&lt;p&gt;Jede Kleinigkeit hilft. Tiled hat sehr viele Benutzer. Würde jeder jeden Monat eine kleine Spende beitragen hätte ich Zeit sicherzugehen, dass Tiled immer besser wird.&lt;/p&gt;
+&lt;p&gt;&lt;i&gt;Thorbj&amp;oslash;rn Lindeijer&lt;/i&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
Comment (Ablu) [inline translations/tiled_de.ts]: I am not sure about the part above... I do not really like it overall... But not sure how to make it better without totally breaking with the english source

Comment (oberhamsi) [inline translations/tiled_de.ts]: maybe like this (deviating more with the english source):

"Mit Deiner Hilfe kann ich weitermachen Tiled zu verbessern!
Vielleicht möchtest du mich finanziell unterstützen. Deine Hilfe würde für mich einen großen Unterschied machen. Ich könnte als Hauptentwickler mehr Zeit in Tiled stecken und müsste weniger andere Jobs annehmen.
<rest same as before>"

**Comment by bjorn** [comment]
Thanks for the update!

I'll ask my wife about the German version of this Patreon dialog as well.

**Comment by bjorn** [comment]
I'll merge this and consider ways to improve the translation of the Patreon dialog later. Thanks!

File: translations/tiled_de.ts (hunk 1 above)
Comment (bjorn) [inline translations/tiled_de.ts]: Now I forgot to look into improving this translation for the release, oh well. I'll try to remember before releasing Tiled 0.12.1. :-)

**Pull request merged**